tachyon/crypto/commitments/kzg/kzg.h

```diff
@@ -89,8 +90,14 @@ class KZG {
     CHECK_EQ(error, gpuSuccess);
     stream_ = device::gpu::CreateStream();
 
+    size_t bases_size = g1_powers_of_tau_.size();
     msm_gpu_.reset(
         new math::VariableBaseMSMGpu<G1Point>(mem_pool_.get(), stream_.get()));
+    d_g1_powers_of_tau_ = device::gpu::GpuMemory<G1Point>::MallocFromPoolAsync(
```

Comment:
What does the "d" prefix stand for?

Reply:
I think it means "device"since we have `device::gpu::GpuMemory<G1Point> d_g1_powers_of_tau_;`

Reply:
@ashjeong  correct!

@TomTaehoonKim When naming to a variable for memory allocated to device(GPU), prefix its name with `d_`. Below is an example of NVIDIA naming in ICICLE:

- [ICICLE example](https://github.com/ingonyama-zk/icicle/blob/main/icicle/src/msm/msm.cu#L413)
- [NVIDIA CUB example](https://github.com/NVIDIA/cub/blob/main/examples/device/example_device_radix_sort.cu#L194-L195)